### PR TITLE
Use updated version of framework

### PR DESCRIPTION
**Pull request by dimos-flexiana**

### CODE REVIEW

File: resources/leiningen/new/xiana/config/dev/config.edn [hunk 1]
```diff
@@ -29,5 +29,23 @@
...
+                                   ;               :over     :all}
+                                   ;              {:resource "posts"
+                                   ;               :actions  [:create :update :delete]
+                                   ;               :over     :own}]
+                                   ;:admin       [{:resource :all
+                                   ;               :actions  [:all]
+                                   ;               :over     :all}]
+                                   }
+
+ }
```
Comment (g-krisztian) [inline resources/leiningen/new/xiana/config/dev/config.edn:51]: Pls add auth config too, from framework's config

File: resources/leiningen/new/xiana/config/test/config.edn [hunk 2]
```diff
@@ -26,7 +26,26 @@
...
+                                   ;               :over     :all}
+                                   ;              {:resource "posts"
+                                   ;               :actions  [:create :update :delete]
+                                   ;               :over     :own}]
+                                   ;:admin       [{:resource :all
+                                   ;               :actions  [:all]
+                                   ;               :over     :all}]
+                                   }
+
+ }
```
Comment (g-krisztian) [inline resources/leiningen/new/xiana/config/test/config.edn:51]: + auth config

File: resources/leiningen/new/xiana/src/backend/app_name.clj [hunk 3]
```diff
@@ -1,41 +1,51 @@
...
-   [reitit.ring :as ring]))
+    [com.stuartsierra.component :as component]
+    [controllers.index :as index]
+    [controllers.re-frame :as re-frame]
+    [framework.components.interceptors :as interceptors]
+    [framework.components.session.backend :as session-backend]
+    [framework.components.web-server.core :as xiana.web-server]
+    [framework.config.core :as config]
+    [framework.db.storage :as db.storage]
+    [nrepl.server :refer [start-server stop-server]]
```
Comment (g-krisztian) [inline resources/leiningen/new/xiana/src/backend/app_name.clj:11]: nrepl server shouldn't be here. Not used, and optional only.

File: resources/leiningen/new/xiana/test/app_name_fixture.clj [hunk 4]
```diff
@@ -23,24 +23,16 @@
     (jdbc/execute! (dissoc db-config :dbname) [init-sql])
     (assoc config :framework.db.storage/postgresql db-config)))
 
-(defn migrate!
-  [config]
-  (let [db (:framework.db.storage/postgresql config)
-        mig-config (assoc (:framework.db.storage/migration config) :db db)]
-    (migratus/migrate mig-config))
-  config)
-
```
Comment (g-krisztian) [inline resources/leiningen/new/xiana/test/app_name_fixture.clj:32]: Just Frankie doesn't use migratus right now, but this should be here at default

File: resources/leiningen/new/xiana/test/app_name_fixture.clj [hunk 5]
```diff
@@ -23,24 +23,16 @@
...
 (defn std-system-fixture
   [f]
-  (let [config (config/edn)
-        system (-> config
+  (let [system (-> (config/edn)
+                   (assoc-in [:framework.app/web-server :port] 3333)
                    embedded-postgres!
-                   migrate!
```
Comment (g-krisztian) [inline resources/leiningen/new/xiana/test/app_name_fixture.clj:38]: let migrate! back as default

File: resources/leiningen/new/xiana/test/app_name_test.clj [hunk 6]
```diff
@@ -11,7 +11,7 @@
 (deftest index-test
   (is (= {:body   "Index page"
           :status 200}
-         (-> {:url                  "http://localhost:3000/"
+         (-> {:url                  "http://localhost:3333/"
               :unexceptional-status (constantly true)
               :method               :get}
              http/request
```
Comment (g-krisztian) [inline resources/leiningen/new/xiana/test/app_name_test.clj:17]: I believe this test is just not working anymore, if the acl-restrict interceptor is executed.

File: resources/leiningen/new/xiana/src/backend/app/interceptors.clj [hunk 7]
```diff
@@ -1,14 +0,0 @@
...
-
-(comment
-  (import-vars
-    [framework.components.app.interceptors
-     sample-router-interceptor
-     sample-controller-interceptor]))
-
-(def sample-{{sanitized-name}}-controller-interceptor
-  {:enter (fn [{request :request {:keys [handler controller match]} :request-data :as state}]
-            (xiana/ok state))})
```
Comment (g-krisztian) [inline resources/leiningen/new/xiana/src/backend/app/interceptors.clj:14]: I don't know, maybe good to have a sample interceptor here, but maybe it should be extended with a :leave fn

**Review by g-krisztian — CHANGES_REQUESTED** [review]

File: resources/leiningen/new/xiana/test/app_name_test.clj (hunk 6 above)
Comment (dimos-flexiana) [reply resources/leiningen/new/xiana/test/app_name_test.clj:17]: In order for the index page to be served, the `acl-restrict` interceptor is disabled (but exist as a comment for letting the user know how to use it). The test therefore works as expected.

File: resources/leiningen/new/xiana/config/dev/config.edn (hunk 1 above)
Comment (dimos-flexiana) [reply resources/leiningen/new/xiana/config/dev/config.edn:59]: https://github.com/Flexiana/xiana-template/blob/ce4fecfe4d3d598c73e2dcb46353b8e1b1440345/resources/leiningen/new/xiana/config/dev/config.edn#L34-L40

File: resources/leiningen/new/xiana/config/test/config.edn (hunk 2 above)
Comment (dimos-flexiana) [reply resources/leiningen/new/xiana/config/test/config.edn:60]: https://github.com/Flexiana/xiana-template/blob/ce4fecfe4d3d598c73e2dcb46353b8e1b1440345/resources/leiningen/new/xiana/config/test/config.edn#L34-L40

File: resources/leiningen/new/xiana/src/backend/app_name.clj (hunk 3 above)
Comment (dimos-flexiana) [reply resources/leiningen/new/xiana/src/backend/app_name.clj]: https://github.com/Flexiana/xiana-template/blob/ce4fecfe4d3d598c73e2dcb46353b8e1b1440345/resources/leiningen/new/xiana/src/backend/app_name.clj#L10-L11

File: resources/leiningen/new/xiana/test/app_name_fixture.clj (hunk 4 above)
Comment (dimos-flexiana) [reply resources/leiningen/new/xiana/test/app_name_fixture.clj]: https://github.com/Flexiana/xiana-template/blob/ce4fecfe4d3d598c73e2dcb46353b8e1b1440345/resources/leiningen/new/xiana/test/app_name_fixture.clj#L27-L32

File: resources/leiningen/new/xiana/test/app_name_fixture.clj (hunk 5 above)
Comment (dimos-flexiana) [reply resources/leiningen/new/xiana/test/app_name_fixture.clj:38]: https://github.com/Flexiana/xiana-template/blob/ce4fecfe4d3d598c73e2dcb46353b8e1b1440345/resources/leiningen/new/xiana/test/app_name_fixture.clj#L38-L40

File: resources/leiningen/new/xiana/src/backend/app/interceptors.clj (hunk 7 above)
Comment (dimos-flexiana) [reply resources/leiningen/new/xiana/src/backend/app/interceptors.clj]: https://github.com/Flexiana/xiana-template/blob/ce4fecfe4d3d598c73e2dcb46353b8e1b1440345/resources/leiningen/new/xiana/src/backend/app/interceptors.clj#L5-L9

**Pull request merged**